pvlib/tests/test_atmosphere.py

```diff
@@ -88,6 +88,143 @@ def test_gueymard94_pw():
     assert_allclose(pws, expected, atol=0.01)
 
 
+# Unit tests
+def test_rh_from_tdew():
+
+    # dewpoint temp calculated with who and aekr coefficients
```

Comment:
several comments and variables in these two test functions say "who" instead of "wmo" :)